inferentia/scripts/gen_triton_model.py

```diff
@@ -394,11 +616,24 @@ def create_model_file():
                         directory where script will generate
                         config.pbtxt and model.py''')
     FLAGS, unparsed = parser.parse_known_args()
+    if len(unparsed) > 0:
+        raise Exception("Unrecognized options: {}".format(unparsed))
+
+    if FLAGS.model_type.lower() == 'tensorflow':
```

Comment:
We no longer need lower or else since we have choices
just use
```
if FLAGS.model_type == 'tensorflow':
        is_tensorflow_model = True
else:
        is_tensorflow_model = False
```